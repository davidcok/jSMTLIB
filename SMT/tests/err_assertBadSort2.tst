; assert with wrong sort
(set-logic QF_LIA)
(assert 1)
