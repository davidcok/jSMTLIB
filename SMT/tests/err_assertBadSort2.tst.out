success
(assert 1)
        ^
(error "Expected an expression with Bool sort, not Int")
